Cargo.toml

```diff
@@ -29,7 +29,7 @@ log = "0.4.8"
 env_logger = "0.7.1"
 libc = { version = "0.2.58", optional = true }
 xenctrl = { git = "https://github.com/Wenzel/xenctrl", optional = true }
-xenstore = { git = "https://github.com/Wenzel/xenstore", rev = 'e8627acfe73186eb7cc34385d531058d3c55643d', optional = true }
+xenstore = { git = "https://github.com/Wenzel/xenstore", rev = '728ed1366723ae2b1415e6d25aad35883fb75d55', optional = true }
```

Comment:
Shouldn't you use your newly published `xenstore-rs` crate here?

Reply:
I know !
I started my PR by switching to xenstore-rs and it would fail to build in the CI.

Could be a bug with cargo and git dependencies.

I will anyway switch when using the crate published on crates.io

Reply:
> I will anyway switch when using the crate published on crates.io

That's what I meant. Also, are we doing a clean checkout? Because if there is a cache somewhere, that would explain why the switch to `xenstore-rs` doesn't work.